packages/gatsby-source-contentful/README.md

```diff
@@ -180,6 +180,12 @@ Enable the new [tags feature](https://www.contentful.com/blog/2021/04/08/governa
 
 Learn how to use them at the [Contentful Tags](#contentful-tags) section.
 
+**`contentTypeFilter`** [function][optional] [default: () => true]
+
+Possibility to limit how many contentType/nodes are created in GraphQL. This can limit the memory usage by reducing the amount of nodes created. Useful if you have a large space in Contentful and only want to get the data from certain content types.
```

Comment:
In a comment you wrote:

> It fails if filter is too broad to the point that all content types are removed

So are the instructions + examples here good enough so that people don't run into that problem? Should the docs give more guidance on how to use this feature?

Reply:
@LekoArts this problem only happens in edge-cases, especially when users use the same space for multiple websites/projects and their space becomes very very big.

IMHO the example below is enough to demonstrate how to use it.

What would you suggest to change/add?